kafka-client/src/test/resources/junit-platform.properties

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright 2023 Responsive Computing, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+junit.jupiter.execution.timeout.default=1m
```

Comment:
This seems a bit low. I'd err towards longer timeouts while aiming for faster tests. I guess the other POV is that this forces us to keep tests fast, but its a little inflexible in cases where the test infra is just slow for some reason. We should keep an eye on build times asynchronously.

Reply:
😆 you predicted that this build would fail, didn't you? I'll bump it to 5m